.gitignore

```
@@ -22,3 +22,6 @@ go.work
 
 # hugo binaries
 hugo
+
+# folders
+public
```

```suggestion
/public/
```

First `/` indicates it's in the root of the git repo and the second one indicates that it's a directory.